wn.ttl

```diff
@@ -450,3 +450,29 @@
   rdfs:label "interlingual index"@en ;
   rdfs:comment "links a synset to a target the Interlingual Index"@en ;
   rdfs:subPropertyOf skos:exactMatch .
+
+:pronunciation a owl:ObjectProperty ;
+  rdfs:label "pronunciation"@en ;
+  rdfs:comment "Indicates the pronunciation of a form"@en ;
+  rdfs:domain ontolex:Form ;
+  rdfs:range :Pronunciation .
+
+:variety a owl:DatatypeProperty ;
+  rdfs:label "variety"@en ;
+  rdfs:comment "Indicates the variety of language the pronunciation represents"@en ;
+  rdfs:domain :Pronunciation .
+
+:notation a owl:DatatypeProperty ;
+  rdfs:label "notation"@en ;
+  rdfs:comment "The notation scheme used for a pronunciation"@en ;
+  rdfs:domain :Pronunciation .
+
+:phonemic a owl:DatatypeProperty ;
+  rdfs:label "phonemic"@en ;
+  rdfs:comment "Indicates if the pronunciation is phonemic, in that represents every aspect of the pronunciation and does not omit information that could be assumed for this language"@en ;
```

Comment:
I think this description is for "phonetic" (see [here](http://australianlinguistics.com/speech-sounds/phonemic-vs-phonetic/) for some explanation). In short, phonetic transcriptions may describe a particular utterance or a highly-specific transcription whereas phonemic transcriptions rely somewhat on the phonology of the language to reduce the level of detail. From the linked article, a phonemic transcription of "clean" would be /klin/ while a phonetic one might be [kl̥i:n]. For wordnets a phonemic transcription is probably more useful so the default value of "true" for the "phonemic" property (in the DTD) seems appropriate.

Reply:
Quite, right on the definition. I think the default value for phonemic is `true` in the DTD, right?

Reply:
> I think the default value for phonemic is true in the DTD, right?

That's what I see in this PR, and I agree that it's a good default. But really (with apologies for the flip-flopping) I'm reconsidering having the delimiters `/.../` and `[...]` in the actual transcription. Originally I suggested not having the `/` slashes when I assumed we only cared about phonemic transcriptions, but if we want to allow phonetic ones (and possibly non-IPA transcriptions for which the phonetic/phonemic difference is unstated), why not just drop the attribute and allow such delimiters? It's shorter and probably clearer to someone who knows IPA.

(should I bring this discussion back to #27?)

Reply:
Yes, possibly best to discuss this on the issue